04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,75 @@
+const Product = require('../models/product')
+
+const getAllProductsStatic = async (req, res) => {
+    const result = Product.find({}).sort('-name').limit(2);
+    products = await result;
+    res.status(200).json({ products });
```

Comment:
You named your variable `result` but tried using one called `products`. Those names should match.

```suggestion
    const products = Product.find({}).sort('-name').limit(2);
    products = await result;
    res.status(200).json({ products });
```